tests/PHPStan/Rules/Functions/CallToFunctionParametersRuleTest.php

```diff
@@ -128,6 +128,21 @@ public function testCallToArrayUnique(): void
 		]);
 	}
 
+	public function testCallToDefineCaseInsenstive(): void
+	{
+		if (PHP_VERSION_ID < 80000) {
+			$this->markTestSkipped('Test requires PHP 8.0.');
+		}
+		$this->analyse([__DIR__ . '/data/call-to-define.php'], [
+			[
+				// @todo wrong error, but not sure where to set it? deprecated parameter.
+				// define(): Argument #3 ($case_insensitive) is ignored since declaration of case-insensitive constants is no longer supported
+				'Function define invoked with 3 parameters, 2 required.',
+				3,
+			],
+		]);
```

Comment:
This might related to the fix needed for array_key_exists where it's not so much about a parameter missing, but becoming deprecated or changing. Tossed this up for feedback